src/layouts/EditNavBar.jsx

```diff
@@ -57,9 +56,19 @@ const EditNavBar = ({ match }) => {
     links: [],
     sublinks: [],
   })
-  const [showDeletedText, setShowDeletedText] = useState(true)
   const [deletedLinks, setDeletedLinks] = useState("")
 
+  const {
+    isOpen: isRemovedOpen,
+    onOpen: onRemovedOpen,
+    onClose: onRemovedClose,
+  } = useDisclosure({ defaultIsOpen: true })
+  const {
+    isOpen: isDeleteOpen,
+    onOpen: onDeleteOpen,
+    onClose: onDeleteClose,
+  } = useDisclosure()
+
```

Comment:
similar to before, we should aim to include `modal` into the names for clarity

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/0721a811504d1b7b38d4d6eb61364e11db356000